### DISCUSSION TITLE
Is it possible to support a legacy date format that is not listed in the dropdown in settings?

### DISCUSSION THREAD

**Discussion by blu-mag** [YU000jp/logseq-plugin-legacy-date-format#2] [Ideas]
My existing date format is `"E, d MMMM yyyy"`, but this isn't listed as a format in plugin settings dropdown. I tried editing the .json settings directly but the change doesn't seem to be reflected.

Would it be possible to support other formats, or am I doing something wrong?

**Comment by YU000jp** [comment]
It is possible. The lack of support for them was not intentional, but rather a simple oversight.😅 I released.

**Comment by blu-mag** [comment]
Awesome! Thank you for the fast work on this. I just updated, and I see the new list of formats in the dropdown, but mine is not listed. There are some similar, but not any that match exactly. 

I am able to manually edit and save the settings.json file with my own format, but it gets overwritten by whatever is in the dropdown when I open the dialog to start the conversation.

Hopefully what I am asking for makes sense, please let me know if you need further clarification.

**Reply by YU000jp** [reply]
I released it in support of that date format. Sorry.

**Reply by blu-mag** [reply]
thank you so much. it worked perfectly!
